Amplifier_Operational.dcm

```diff
@@ -1356,6 +1356,18 @@ K single opamp wideband low-power
 F http://www.ti.com/lit/ds/symlink/opa847.pdf
 $ENDCMP
 #
+$CMP OPA855
```

Comment:
OPA855xDSG

Name should contain package suffix, but not temp rating, packaging information and such.